### PR TITLE
Adds a couple mood events

### PR DESCRIPTION
**Pull request by wesoda25**
## About The Pull Request

Being put in an electric chair/guillotine now gives you a terrible mood debuff. Being held up also gives you a bad mood debuff, but not as bad. (note: neither are lasting, so shitters cant point a laser tag gun at you or strap you into a e chair to fuck with your mood)

## Why It's Good For The Game

Adds more flavor to small things in the game, better IMMERSION

## Changelog
:cl:
add: Adds a couple new mood events for being held up and put in an electric chair/guillotine.
/:cl:

### CODE REVIEW

**Comment by tralezab** [comment]
great small edition as per usual ol' pal

**Comment by wesoda25** [comment]
Thanks bro. Also I just realized guillotines could use the "about to die" one as well so I'll add it to them too.

**Comment by ATH1909** [comment]
> great small edition as per usual ol' pal

*addition

File: code/datums/mood_events/generic_negative_events.dm [hunk 1]
```diff
@@ -212,6 +212,14 @@
 	mood_change = -8
 	timeout = 3 MINUTES
 
+/datum/mood_event/deaths_door
+	description = "<span class='boldwarning'>This is it... I'm really going to die.</span>\n"
+	mood_change = -20
+
+/datum/mood_event/gunpoint
+	description = "<span class='boldwarning'>This guy is insane! I better be careful....</span>\n"
```
Comment (ATH1909) [inline code/datums/mood_events/generic_negative_events.dm]: police officer: *points a disabler at you and asks you kindly to come quietly*
gunpoint moodlet: "This guy is INSANE!"
maybe make the gunpoint moodlet only trigger when someone points a LETHAL weapon at you?

**Review by kriskog — APPROVED** [review]

**Pull request merged**